docs/backplane/guideline.md

```diff
@@ -0,0 +1,103 @@
+
+In general all permissions must follow the principle of least privilege.  Meaning you only get the bare minimum permissions required to do the job, nothing more.
+
+## Training
+
+This guideline assumes all personas with access to production customer clusters have been through all necessary vetting (background checks) and training before access is granted.
+
+## Access Scope
+
+A persona must have access only to systems on which support can actively be provided by that persona.  This means if a presona is dependent on some software being installed, a layered product / addon / managed service, then the persona is not permitted access to the system until that installation is triggered.  Note this means access is not dependent on successfully deploying that software!
+
+## Personas
+
+* Managed Cloud Service (MCS)
+   * formerly Customer Experience and Engagement (CEE)
+* Site Reliability Engineer (SRE)
+   * https://landing.google.com/sre/ 
+* Platform SRE (SREP)
+   * SRE in SD primarily responsible for Managed OpenShift Platform
+   * https://source.redhat.com/groups/public/sre 
+* Layered Product SRE (LPSRE)
+   * SRE in SD primarily responsible for Managed Services / Layered Products
+   * formerly MT-SRE and CS-SRE
+   * https://source.redhat.com/groups/public/sre-services/sre_services_wiki/managed_tenants_sre_introduction
+   * https://source.redhat.com/groups/public/sre-services/sre_services_wiki/cssre_introduction 
+* Application SRE (AppSRE)
+   * SRE in SD primarily responsible for Red Hat internal applications and capabilities provided to other Red Hat personas and Red Hat customers
+   * https://source.redhat.com/groups/public/sre-services/sre_services_wiki/app_sre_introduction_new 
+* Hybrid SRE
+   * Engineering teams are SRE for their products in conjunction with other SD SRE teams.
+   * https://source.redhat.com/groups/public/sdsea/hybridsre 
+   * Teams that are the initial focus include HyperShift, ACS, and ODS
+
+## Privilege Escalation
+
+Some personas may elevate permissions beyond the day-to-day permisisons.  All such elevations are audited and require justification and approval from managers after use and must never be used lightly.
+
+Personas permitted to use privilege escalation on customer clusters are:
+* SREP
+* LPSRE
+
+Escalation of privileges uses a specific user to eleavte permissios to `cluster-admin`.  More information on [ops-sop/v4/howto/backplane-elevate-privileges.md](https://github.com/openshift/ops-sop/blob/master/v4/howto/backplane-elevate-privileges.md).
+
+## Day-to-Day
+
+Permissions for day-to-day operations should not require any special permission to utilize.  These permissions follow these guidelines:
+
+* No secret access except where the secret is known to contain no actual secret data.
+* No customer namespace access as the baseline.  Customer namespace access must be justified based on specific service support needs.
```

Comment:
Is this available today without privilege escalation?

Reply:
Access to customer namespaces requires impersonating `cluster-admin` today.

Reply:
I propose we have a separete section detailing when you can use the privilege escalation process and include customer namespaces in there.

Reply:
The [SOP](https://github.com/openshift/ops-sop/blob/master/v4/howto/backplane-elevate-privileges.md#trust-and-verify) generally describes the conditions under which you can impersonate and what the consequences are. I think the term _customer_ namespace is actually the issue here. We should be saying _unprotected_ namespace i.e. namespaces not prefixed with `redhat-`, `openshift-`, or `kube-`. Namespaces which are not protected by MCVW pose a risk in the form of cross-contamination with customer data and could require SRE's to operate alongside the customers. RHODS is an example of this with the `rhods-notebooks` namespace.

True customer namespaces which are unrelated to managed services should never be touched by SRE and when accessed via impersonation should trigger an unjustifiable audit event.